dev/release/release.sh

```diff
@@ -57,7 +57,7 @@ gh release download "${rc_tag}" \
 
 echo "Uploading to release/"
 pushd "${dist_base_dir}"
-svn add .
+svn add "${release_id}"
```

Comment:
Oh. I thought that "." also works.
Do you still have an actual error message?

Reply:
yea, I get

```console
svn: warning: W150002: '/tmp/arrow-go/dev/release/dist' is already under version control
svn: E200009: Could not add all targets because some targets are already versioned
svn: E200009: Illegal target for the requested operation
```

when I use ".", which is fixed if I use the directory name

Reply:
Thanks!

I'll fix related codes in apache/arrow-java.

Reply:
https://github.com/apache/arrow-java/pull/513